src/views/studio/CreateEditChannelView/CreateEditChannelView.tsx

```diff
@@ -350,25 +350,52 @@ const CreateEditChannelView: React.FC<CreateEditChannelViewProps> = ({ newChanne
   const hasCoverUploadFailed = channel?.coverPhotoAvailability === AssetAvailability.Pending
 
   return (
-    <>
-      <DataLostWarningDialog />
-      <form onSubmit={handleSubmit}>
-        <Header>
+    <form onSubmit={handleSubmit}>
+      <Header>
+        <Controller
+          name="cover"
+          control={control}
+          render={({ value, onChange }) => (
+            <>
+              <ChannelCover
+                coverPhotoUrl={loading ? null : value.url}
+                onCoverEditClick={() => coverDialogRef.current?.open()}
+                onCoverRemoveClick={() => onChange({ blob: null, url: null })}
+                editable
+                disabled={loading}
+              />
+              <ImageCropDialog
+                imageType="cover"
+                onConfirm={(blob, url, assetDimensions, imageCropData) =>
+                  onChange({ blob, url, assetDimensions, imageCropData })
+                }
+                onError={() =>
+                  displaySnackbar({
+                    title: 'Cannot load the image. Choose another.',
+                    iconType: 'error',
+                  })
+                }
+                ref={coverDialogRef}
+              />
+            </>
+          )}
+        />
+
+        <StyledTitleSection className={transitions.names.slide}>
           <Controller
-            name="cover"
+            name="avatar"
             control={control}
             render={({ value, onChange }) => (
               <>
-                <ChannelCover
-                  coverPhotoUrl={loading ? null : value.url}
-                  hasCoverUploadFailed={hasCoverUploadFailed}
```

Comment:
This shouldn't be removed?

Reply:
The diff here is a bit broken because of whitespace changes but `hasCoverUploadFailed` line seems to be removed